charts/tornjak/templates/server-statefulset.tpl

```diff
@@ -0,0 +1,157 @@
+  replicas: 1
+  selector:
+    matchLabels:
+      app: spire-server
+  serviceName: spire-server
+  template:
+    metadata:
+      namespace: {{ .Values.namespace }}
+      labels:
+        app: spire-server
+    spec:
+      serviceAccountName: spire-server
+      shareProcessNamespace: true
+      containers:
+        - name: spire-server
+          # image: gcr.io/spiffe-io/spire-server:0.11.0
+          image: {{ .Values.tornjakImg }}:{{ .Values.spireVersion }}
+          imagePullPolicy: Always
+          args:
+            - -config
+            - /run/spire/config/server.conf
+          ports:
+            - containerPort: 8081
+          securityContext:
+            # privileged is needed to access mounted files (e.g. /run/spire/data)
+            # not needed if using volumeClaimTemplates and sockets
+            privileged: true
+          volumeMounts:
+            - name: spire-config
+              mountPath: /run/spire/config
+              readOnly: true
+            - name: spire-data
+              mountPath: /run/spire/data
+              readOnly: false
+            - name: certs
+              mountPath: /opt/spire/sample-keys
+            - name: spire-server-socket
+              mountPath: /run/spire/sockets
+              readOnly: false
+          livenessProbe:
+            exec:
+              command:
+              - "/opt/spire/bin/spire-server"
+              - "healthcheck"
+              - "-registrationUDSPath"
+              - "{{ .Values.spireServerSocket }}"
+            failureThreshold: 2
+            initialDelaySeconds: 15
+            periodSeconds: 60
+            timeoutSeconds: 3
+{{- if .Values.OIDC.enable }}
+          readinessProbe:
+            exec:
+              command:
+              - "/opt/spire/bin/spire-server"
+              - "healthcheck"
+              - "-registrationUDSPath"
+              - "{{ .Values.spireServerSocket }}"
+              - "--shallow"
+            initialDelaySeconds: 5
+            periodSeconds: 10
+{{- end }}
+{{- if .Values.OIDC.enable }}
+        - name: spire-oidc
+          image: gcr.io/spiffe-io/oidc-discovery-provider:{{ .Values.spireVersion }}
+          args:
+          - -config
+          - /run/spire/oidc/config/oidc-discovery-provider.conf
+          ports:
+          - containerPort: 443
+            name: spire-oidc-port
+          volumeMounts:
+          - name: spire-server-socket
+            mountPath: /run/spire/sockets
+            readOnly: true
+          - name: spire-oidc-config
+            mountPath: /run/spire/oidc/config/
+            readOnly: true
+          - name: spire-data
+            mountPath: /run/spire/data
+            readOnly: false
+          - name: spire-oidc-socket
+            mountPath: /run/oidc-discovery-provider/
+          readinessProbe:
+            exec:
+              command: ["/bin/ps", "aux", " ||", "grep", "oidc-discovery-provider -config /run/spire/oidc/config/oidc-discovery-provider.conf"]
```

Comment:
not sure how this is evaluated, would like to check this, since the pipe is a bash construct (and this one is an OR as well maybe)